File: _프로그래머스/01-20/17. 수박수박수박수.py
"""
* Title: 수박수박수박수
https://school.programmers.co.kr/learn/courses/30/lessons/12922
"""
def solution(n):
    answer = ('수박' * (n//2))
    if n % 2 == 1:
        answer += '수'

    return answer

File: _프로그래머스 js/17. 수박수박수박수.js
/*
* Title: 수박수박수박수
https://school.programmers.co.kr/learn/courses/30/lessons/12922
*/
function solution(n) {
  let val = '수박'.repeat(Math.floor(n / 2));

  if (n % 2 === 1) {
    val += '수';
  }

  return val;
}

// "수박수"
console.log(solution(3));
